source/external_server_api.cpp

```diff
@@ -268,10 +275,6 @@ int wait_for_command(int timeout_time_in_ms, void *context) {
             received_device_id->getRole(),
             received_device_id->getName()
         ));
-
-        if(command->getTimestamp() > con->last_command_timestamp) {
```

Comment:
Why this is deleted?

Reply:
It's moved earlier in the function because I realized that the timestamp needs to be checked before parsing the protobuf command in case it fails